tests/linked_list_tests.py

```diff
@@ -0,0 +1,142 @@
+import unittest
+from linked_list.linked_list import LinkedList
+
+
+class TestLinkedList(unittest.TestCase):
+
+    def test_attempt_to_add_none_value_raises_type_error(self):
+        linkedlist = LinkedList()
+        try:
+            linkedlist.append(None)
+            self.fail("Should be an exception")
+        except TypeError as e:
+            self.assertEqual("Value is absent!", str(e))
+
+    def test_possible_to_add_negative_value(self):
+        linkedlist = LinkedList()
+        linkedlist.append(-2.89)
+        self.assertEqual(1, linkedlist.length_of_list)
+
+    def test_possible_to_add_zero_value(self):
+        linkedlist = LinkedList()
+        linkedlist.append(0)
+        self.assertEqual(1, linkedlist.length_of_list)
+
+    def test_possible_to_add_text_value(self):
+        linkedlist = LinkedList()
+        linkedlist.append('test')
+        self.assertEqual(1, linkedlist.length_of_list)
+
+    def test_possible_to_add_several_values(self):
+        linkedlist = LinkedList()
+        linkedlist.append('test')
+        linkedlist.append('01test')
+        self.assertEqual(2, linkedlist.length_of_list)
+
+    def test_length_of_list_is_0_in_case_first_node_is_none(self):
```

Comment:
`first_node_is_none` is implementation details. It's possible that you'll refactor the code some day and it won't be using nodes or will work differently in some other way. The outside world only knows that it didn't add any elements. Hence the name should be something like `test_length_is_0_if_no_elements_have_been_added`